base/regex.jl

```diff
@@ -7,6 +7,16 @@ include("pcre.jl")
 const DEFAULT_COMPILER_OPTS = PCRE.UTF | PCRE.NO_UTF_CHECK | PCRE.ALT_BSUX | PCRE.UCP
 const DEFAULT_MATCH_OPTS = PCRE.NO_UTF_CHECK
 
+"""
+    Regex(pattern[, flags])
+
+A type representing a regular expression. `Regex`-objects can be used to match strings
```

Comment:
Why the hyphen? I don't think it's warranted.

Reply:
thanks for the comment!